qa/tasks/cephfs/test_misc.py

```diff
@@ -290,6 +291,71 @@ def test_file_sync_stuck_for_around_5s(self):
         dir_path = "file_sync_do_not_wait_mdlog_testdir"
         self._test_sync_stuck_for_around_5s(dir_path, True)
 
+    def test_file_filesystem_sync_crash(self):
+        """
+        To check whether will the kernel crash when doing the file/filesystem sync.
+        """
+
+        stop_thread = False
+        dir_path = "file_filesystem_sync_crash_testdir"
+        self.mount_a.run_shell(["mkdir", dir_path])
+
+        def mkdir_rmdir_thread(mount, path):
+            #global stop_thread
+
+            log.info(" mkdir_rmdir_thread starting...")
+            num = 0
+            while not stop_thread:
+                n = num
+                m = num
+                for i in range(10):
```

Comment:
`for __ in range(10):` is preferred if `variable` is not used. https://docs.python-guide.org/writing/style/#create-an-ignored-variable

Reply:
Okay, will fix it.

Reply:
This is nit. Looks good to me.

Reply:
Sorry, I thought I have fixed it. will push it again.